stdlib/LinearAlgebra/src/bidiag.jl

```diff
@@ -202,6 +202,80 @@ function svd(M::Bidiagonal; full::Bool = false)
     svd!(copy(M), full = full)
 end
 
+###################
+#     opnorms     #
+###################
+
+# helpers for upper and lower bidiagonal matrices
+
+function _upopnorm1(A::Bidiagonal{T}) where T
+    n = size(A, 1)
+    Tnorm = typeof(float(real(zero(T))))
+    Tsum = promote_type(Float64, Tnorm)
+    nrm::Tsum = A.dv[1]
+    @inbounds begin
+        for i = 2:n
+            nrmj::Tsum = norm(A.dv[i]) + norm(A.ev[i-1])
+            nrm = max(nrm,nrmj)
+        end
+    end
+    return convert(Tnorm, nrm)
+end
+
+function _loopnorm1(A::Bidiagonal{T}) where T
```

Comment:
lower bidiagonal p=1 norm is the same code as upper bidiagonal p=Inf (and the same with upper p=1 and lower p=Inf). These can be combined but that would reduce readability in my opinion. Thoughts on how to make this cleaner?

One option is to do `_upnormInf(A::Bidiagonal) = _loopnorm1(A)` and `_lonormInf(A::Bidiagonal) = _upnorm1(A)` and document it well.